lib/quadquizaminos_web/router.ex

```diff
@@ -40,6 +40,7 @@ defmodule QuadquizaminosWeb.Router do
     live "/leaderboard/:board_id", LeaderboardLive.Show
     live "/contestboard", ContestboardLive
     live "/contests", ContestsLive
+    live "/contests/:contest_id", ContestsLive.Show
```

Comment:
We can pass action option on the ContestLive (more like what we have in REST architecture).
Have a look at [these docs](https://hexdocs.pm/phoenix_live_view/Phoenix.LiveView.Router.html#live/4)

Quick snippet
```elixir
live "/articles", ArticleLive.Index, :index
live "/articles/new", ArticleLive.Index, :new
live "/articles/:id/edit", ArticleLive.Index, :edit
```

Reply:
Working on it